.NET/Microsoft.Recognizers.Text.DateTime/Constants.cs

```diff
@@ -81,10 +81,12 @@ public static class Constants
         public static readonly int MaxTwoDigitYearFutureNum = int.Parse(BaseDateTime.MaxTwoDigitYearFutureNum);
         public static readonly int MinTwoDigitYearPastNum = int.Parse(BaseDateTime.MinTwoDigitYearPastNum);
 
-        // special value for timezone
+        // Special values for timezone
         public const int InvalidOffsetValue = -10000;
-        public const string UtcOffsetMinsKey = "UtcOffsetMins";
-        public const string ResolveTimeZone = "resolveTimeZone";
+        public const string UtcOffsetMinsKey = "utcOffsetMins";
+        public const string TimezoneText = "timezoneText";
+        public const string Timezone = "timezone";
+        public const string ResolveTimezone = "resolveTimeZone";

```

Comment:
ResolvedTimezone?

Also, let's document exactly what goes in the single timezone entity, and what goes into the merged datetime entity with timezone.

Reply:
Added in BaseMergedDateTimeParser.